### ISSUE TITLE
如何让webfont支持transform属性

### ISSUE THREAD

**Issue by zhuangweizhan** [itgalaxy/webfont#612]
<use transform="matrix(1 0 0 -1 0 21.985684)" xlink:href="#8drokg0ad__cwrtvxita__medt7g7dka"></use>
如何让webfont支持transform属性

**Comment by jimmyandrade** [comment]
Thanks for the example — this is a known limitation of the SVG → font pipeline, not a missing `transform` flag on webfont.

## Cause

**svgicons2svgfont** (used internally) converts filled paths per icon file. It does **not**:

- resolve **`<use xlink:href="#…">`** / `<symbol>` instances, or
- bake **`transform`** on those references into standalone path data.

Your snippet:

```xml
<use transform="matrix(1 0 0 -1 0 21.985684)" xlink:href="#…"></use>
```

renders in browsers but often produces an **empty or wrong glyph** in the font.

`transform` on normal paths/groups is only **experimental** upstream; `<use>` is a separate gap.

## Workaround (works today on 12.1.0)

1. **Flatten** in your design tool (Expand / Flatten) so each icon is plain paths.
2. Or run **SVGO** (or similar) in `glyphContentTransformFn` before conversion.
3. Scan with **`webfont icons/*.svg --svg-diagnose`** — PR #733 adds a **`use-reference`** warning when `<use>` is detected.

## Docs

- [TROUBLESHOOTING.md — SVG transform and `<use>` references](https://github.com/itgalaxy/webfont/blob/master/TROUBLESHOOTING.md) (after #733 merges)
- [`docs/migration/issue-0612-svg-use-transform.md`](https://github.com/itgalaxy/webfont/blob/master/docs/migration/issue-0612-svg-use-transform.md)

We are **not** adding automatic `<use>` inlining inside webfont (out of scope for the core library); diagnostics + docs are the supported path. Leaving the issue open until #733 merges; we can close after the next npm release documents the diagnostic.
